<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibreMinesPreferencesDialog</class>
 <widget class="QDialog" name="LibreMinesPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbCheckBoxes">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>267</width>
     <height>193</height>
    </rect>
   </property>
   <property name="title">
    <string>On Game Options</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="cbFirstCellClean">
      <property name="text">
       <string>First Cell Clean</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbCleanNeighborCellsWhenClickedOnShowedCell">
      <property name="text">
       <string>Clean Neighbor Cells When
Clicked On Showed Cell</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="labelMinimumCellLength">
        <property name="text">
         <string>Minimum Cell Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbMinimumCellLength">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="labelMaximumCellLength">
        <property name="text">
         <string>Maximum Cell Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbMaximumCellLength">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>999</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbLineEdits">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>9</y>
     <width>273</width>
     <height>75</height>
    </rect>
   </property>
   <property name="title">
    <string>User Information</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelUsername">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditUsername"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbAppearance">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>280</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Appearance</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelApplicationTheme">
        <property name="text">
         <string>Application Theme:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxApplicationTheme"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="labelMinefieldTheme">
        <property name="text">
         <string>Minefield Theme:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxMinefieldTheme"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>374</width>
     <height>285</height>
    </rect>
   </property>
   <property name="title">
    <string>Keyboard Controller</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Move Current Cell Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeyInput" name="keyInputMoveLeft">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Move Current Cell Upwards</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeyInput" name="keyInputMoveUp">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Move Current Cell Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeyInput" name="keyInputMoveRight">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Move Current Cell Downwards</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeyInput" name="keyInputMoveDown">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Release Current Cell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeyInput" name="keyInputReleaseCell">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>(Un)flag Current Cell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QKeyInput" name="keyInputFlagCell">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>When Ctrl is pressed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxWhenCtrlIsPressed"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeyInput</class>
   <extends>QLineEdit</extends>
   <header>qkeyinput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
